docs/CompilerPerformance.md

```diff
@@ -0,0 +1,1083 @@
+# Swift Compiler Performance
+
+This document is a guide to:
+  - Understanding Swift compilation performance
```

Comment:
"Understanding Swift compilation performance" is very nebulous especially since the title of the document is Swift Compiler Performance. A specific title would be really useful here for the reader.

Reply:
Changed to 'Outline of processes and factors affecting compilation performance'.